content/4_Gold/Modular.mdx

```diff
@@ -296,19 +316,19 @@ We can also use the extended Euclidean algorithm. See the module in the
 	</Resource>
 </Resources>
 
-Using my template, both of these do the same thing:
+Using BenQ's template, both of these do the same thing:
 
 ```cpp
 int main() {
 	{
 		int a = 1e8, b = 1e8, c = 1e8;
-		cout << (ll)a*b%MOD*c%MOD << "\n"; // 49000000
+		cout << (ll) a * b % MOD * c % MOD << "\n"; // 49000000
 	}
 	{
 		mi a = 1e8, b = 1e8, c = 1e8;
```

Comment:
maybe the template could be added as a codesnip?

Reply:
You're not talking about the entire benq template right?

Reply:
just the modInt one?